### ISSUE TITLE
CI fails for PHP 7.3 and 7.4

### ISSUE THREAD

**Issue by phil-davis** [sabre-io/http#157]
See demo PR #156 
And Travis logs:
https://travis-ci.org/github/sabre-io/http/jobs/732214024
```
$ if [ $RUN_PHPSTAN == "FALSE" ]; then php vendor/bin/phpunit --configuration tests/phpunit.xml $WITH_COVERAGE; fi

PHPUnit 9.4.0 by Sebastian Bergmann and contributors.

Warning:       Your XML configuration validates against a deprecated schema.

Suggestion:    Migrate your XML configuration using "--migrate-configuration"!

.............................F.................................  63 / 158 ( 39%)

............................................................... 126 / 158 ( 79%)

................................                                158 / 158 (100%)

Time: 00:13.094, Memory: 56.00 MB

There was 1 failure:

1) Sabre\HTTP\ClientTest::testSendToGetLargeContent

Failed asserting that 41943040 is greater than 49558920.

/home/travis/build/sabre-io/http/tests/HTTP/ClientTest.php:215

FAILURES!

Tests: 158, Assertions: 288, Failures: 1.
```
https://travis-ci.org/github/sabre-io/http/jobs/732214026

**Comment by phil-davis** [comment]
A previous green CI was https://travis-ci.org/github/sabre-io/http/jobs/648231485 which had PHPunit 9.0.0

**Comment by phil-davis** [comment]
Fixed by PR #158
